controllers/login.js

```diff
@@ -17,6 +17,9 @@ const getSelectOptions = (req, service, query) => api(req).get(`/${service}`, {
 }).then(data => data.data);
 
 const clearCookie = (req, res) => {
+	if (req.session && req.session.destroy) {
```

Comment:
How is this set?

Reply:
I think it's more some kind of check, if the method exists and can be executed.
Maybe it's needed if the session should get deleted but isn't yet created for the current user?
@tofixx